src/components/navigation/perPage/NavigationPerPageConnected.tsx

```diff
@@ -34,7 +34,7 @@ const mapDispatchToProps = (dispatch: (action: IReduxAction<IReduxActionsPayload
     onPerPageClick: (perPageNb: number, oldPerPageNb: number, currentPage: number) => {
       dispatch(turnOnLoading(ownProps.loadingIds));
       dispatch(changePerPage(ownProps.id, perPageNb));
-      dispatch(changePage(`pagination-${ownProps.id}`, Math.floor(currentPage * oldPerPageNb / perPageNb)));
+      dispatch(changePage(`pagination-${ownProps.id}`, 0));
```

Comment:
Any particular reason why this was changed?

Reply:
I thought this was more consistent with other behaviors coming from other components (onFilter resets the page to 0, onSort resets the page to 0, onPredicate resets the page to 0)...after we discussed i'll probably revert to the old behavior somehow.